app/views/vm_common/_associate_floating_ip.html.haml

```diff
@@ -8,26 +8,25 @@
         = _('Floating IP')
       .col-md-8
         %select{:name      => 'floating_ip',
-                'ng-model' => 'vmCloudModel.floating_ip',
-                'ng-options' => 'floating_ip.address for floating_ip in floating_ips track by floating_ip.address'}
+                'ng-model' => 'vm.vmCloudModel.floating_ip',
+                'ng-options' => 'floating_ip.address for floating_ip in vm.floating_ips track by floating_ip.address'}
 
-  %div_for_paging{'ng-controller'                    => "pagingDivButtonGroupController",
-                  'paging_div_buttons_id'            => "angular_paging_div_buttons",
-                  'paging_div_buttons_type'          => "Submit"}
+    %div_for_paging{'ng-controller'                    => "pagingDivButtonGroupController",
```

Comment:
Probably should not live inside `.form_horizontal`, right?

Or rather, either moving this was a mistake, or you want to do this in the other 8 templates using it as well...